apis/accesskey/v1alpha1/types.go

```diff
@@ -20,36 +20,63 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 
 	xpv1 "github.com/crossplane/crossplane-runtime/apis/common/v1"
+	"github.com/crossplane/provider-bitbucket-server/internal/clients/bitbucket"
 )
 
 // AccessKeyParameters are the configurable fields of a AccessKey.
 type AccessKeyParameters struct {
-	ConfigurableField string `json:"configurableField"`
+	// +immutable
+	ProjectName string `json:"projectName"`
+
+	// +immutable
+	RepoName string `json:"repoName"`
```

Comment:
`RepoName` is referd to as `RepoSlug` in bitbucket API ref and doc, as it is url sanitized. But it is often just the reponame, so details :)

Reply:
I'm not sure it this would be better for the user, because the UI simply calls it "Name" / "repository name".